keadm/cmd/keadm/app/cmd/util/image.go

```diff
@@ -154,7 +154,7 @@ func (runtime *DockerRuntime) RemoveMQTT() error {
 		All: true,
 	}
 	options.Filters = filters.NewArgs()
-	options.Filters.Add("ancestor", constants.DefaultMosquittoImage)
```

Comment:
Thanks for the fixing, and could I please ask what's the meaning of `ancestor` here?

Reply:
This condition `ancestor` uses image to filter containers.  But we use a constant of image when we reset node.  When we use a custom image repository for join node, it causes changes to the mqtt image. This will result in the mqtt container to not be deleted.

Reply:
yeah, use the special image name only can find special contained,  when we use anthoer image to replace the mosqutto,  the condition `ancestor` can't be work normally,  At the same time, the name of mqtt container is unique, and whatever image you used, it always could find the mqtt container and remove it.